src/native/external/libunwind.cmake

```diff
@@ -1,7 +1,10 @@
-# This is a custom file written for .NET Core's build system
-# It overwrites the one found in upstream
+include_directories(${CMAKE_CURRENT_LIST_DIR}/libunwind/include/tdep)
```

Comment:
This file is missing the license header.

Reply:
Looks like we usually do not have license headers in cmake files. Out of roughly 300 cmake files in the repo, only 30 has the license header:
```zsh
#  grand total: cmake files
% git ls-files ':/*.cmake' ':/*CMakeLists.txt' | wc -l
     304

# number of files with the license headers
% git grep 'The .NET Foundation' ':/*.cmake' ':/*CMakeLists.txt' | wc -l
      30
```
all 30 of them are under `src/native/corehost`.